multicluster/apis/multicluster/v1alpha1/clusterset_webhook.go

```diff
@@ -45,12 +47,26 @@ var _ webhook.Validator = &ClusterSet{}
 // ValidateCreate implements webhook.Validator so a webhook will be registered for the type
 func (r *ClusterSet) ValidateCreate() error {
 	klog.InfoS("validate create", "name", r.Name)
+	size := len(r.Spec.Leaders)
```

Comment:
We may use openAPIV3Schema minItems / maxItems to do the validation. Check: #3920

Reply:
Done. ClusterSet webhook is removed since we don't need it.